crates/database/src/states/reverts.rs

```diff
@@ -198,6 +236,115 @@ impl AccountRevert {
     }
 }
 
+/// Implements partial ordering for AccountInfoRevert
+impl PartialOrd for AccountInfoRevert {
+    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+/// Implements total ordering for AccountInfoRevert
+impl Ord for AccountInfoRevert {
+    fn cmp(&self, other: &Self) -> Ordering {
+        self.partial_cmp(other).unwrap()
```

Comment:
also wondering about this unwrap and whether or not we can just use `derive(PartialOrd, Ord)` on `AccountInfo Revert`